### ISSUE TITLE
[D9] Add menu link support into views

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4505]
This was brought up by @BWPanda in Zulip:

>BWPanda (Peter): Anyone know how to get details about a node's menu link in views in D7? E.g. I have a view of nodes, and alongside the node title, body, etc. I want to output details about that node's menu link (like the menu link name, parent, weight, etc.) Is that possible?
>...
>Ok, looks like I can use https://www.drupal.org/project/menu_node_views

>The Menu Node Views module provides Views integration for nodes assigned to the menu system. ... This module consolidates and replaces work done by both http://drupal.org/project/node_menu and http://drupal.org/project/views_menu_nodes.

...and it seems to be on the roadmap for D9 core Views: https://www.drupal.org/project/drupal/issues/2777749

**Comment by BWPanda** [comment]
Makes sense to add to core I think. We currently have support for published status, posted/updated date/time, author, etc. So adding menu details makes sense to me.
